iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -826,4 +866,31 @@ private StringBuilder buildComplexTypeInnerQuery(Object field, Type type) {
     }
     return query;
   }
+
+  private void validateBasicStats(Table table, String tableName) {
+    List<Object[]> describeResult = shell.executeStatement("DESCRIBE EXTENDED " + tableName);
+    Optional<Object[]> tableInfo =
```

Comment:
Instead of using a `describe` command, wouldn't it be cleaner to load the HMS table and check its HMS params, then we wouldn't need all the parsing. It's done in a similar fashion here: https://github.com/apache/iceberg/blob/master/mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java#L585

Reply:
Thanks for letting me know. I changed the test.